Error: can't find class file while running the class file on java

Following the tutorial, after running VOC over example.py, the next command i.e.
`$ java -classpath ../voc/dist/python-java-support.jar:. python.example`
gives the following error:

```
Exception in thread "main" java.lang.ExceptionInInitializerError
        at python.sys.<clinit>(sys.java:76)
	at python.example.main(example.py)
Caused by: RuntimeError: Unknown platform vendor 'Azul Systems, Inc.'
	at python.platform.<clinit>(platform.java:29)
	... 2 more
```

When used with `-v`, it gives:
`Error: Could not find or load main class ...voc.dist.python-java-support.jar:.`

But the file "python-java-support.jar" exist at the given location.

--- a/python/common/python/platform.java
+++ b/python/common/python/platform.java
@@ -25,7 +25,9 @@ public class platform extends org.python.types.Module {
             platform_class_name = "python._platform.JavaPlatform";
         } else if (vendor.equals("The Android Project")) {
             platform_class_name = "python._platform.AndroidPlatform";
-        } else {
+        } else if (vendor.equals("Azul Systems, Inc.")) {
+	    platform_class_name = "python._platform.JavaPlatform";
+	} else {
             throw new org.python.exceptions.RuntimeError("Unknown platform vendor '" + vendor + "'");
         }
 

